src/js/components/AnalysisPanel/Analysis.js

```diff
@@ -29,6 +33,8 @@ import React, {
   Component,
   PropTypes
 } from 'react';
+import analysisUtils from '../../utils/analysisUtils';
```

Comment:
Let's not import the whole analysisUtils here if we are doing specific functions From it (line 24). We can just choose to import the methods we need or the whole thing.

Reply:
I think this was an auto import from my editor. I will fix it!